rholang/src/test/scala/coop/rchain/rholang/interpreter/util/RevAddressSpec.scala

```diff
@@ -17,4 +17,17 @@ class RevAddressSpec extends FlatSpec with Matchers {
     )
   }
 
+  "fromEthAddress" should "work correctly without a prefix" in {
+    val ethAddress = "06a441c277bf454c5d159b0e5bdafca69b296733"
+    RevAddress.fromEthAddress(ethAddress).map(_.toBase58) should be(
+      Some("1111Gzo7ywxbcXVumSS9Lzd8JBAqnF1zniNszvMLHQ2APa3dzs2rG")
+    )
+  }
+
+  "fromEthAddress" should "work correctly with a prefix" in {
+    val ethAddress = "0x06a441c277bf454c5d159b0e5bdafca69b296733"
+    RevAddress.fromEthAddress(ethAddress).map(_.toBase58) should be(
+      Some("1111Gzo7ywxbcXVumSS9Lzd8JBAqnF1zniNszvMLHQ2APa3dzs2rG")
+    )
+  }
```

Comment:
Please add negative cases: wrong alphabet, wrong length

Reply:
We should also probably implement the address checksum checking according to https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md. I'm fine with a follow-up PR for that.

Reply:
Done PTAL